src/couch/src/couch_db.erl

```diff
@@ -293,23 +303,33 @@ delete_doc(Db, Id, Revisions) ->
 open_doc(Db, IdOrDocInfo) ->
     open_doc(Db, IdOrDocInfo, []).
 
-open_doc(Db, Id, Options) ->
+open_doc(Db, Id, Options0) ->
     increment_stat(Db, [couchdb, database_reads]),
+    Options = case has_access_enabled(Db) of
+        true -> Options0 ++ [conflicts];
```

Comment:
Original set of comments https://github.com/apache/couchdb/pull/3038#discussion_r475014140

> I'm confused; why does a DB that has access enabled always get the `conflicts` option added to a doc read? I know only admins can see conflicts, so is this to say "if this document is conflicted, deny all per-user access until an admin can resolve the conflicts?"
>[...]
>this is so we can deny access to a doc if the two edit branches disagree on who should have access, the doc becomes admin-only then.

Reply:
discussion moved here https://github.com/apache/couchdb/pull/4139#discussion_r939533587